src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -1782,4 +1783,39 @@ private static String buildVectorElements(int offset, int count) {
     }
     return sb.toString();
   }
+
+  @Nested
+  @Order(99)
+  class Metrics {
+    @Test
+    public void checkInsertOneMetrics() {
+      VectorSearchIntegrationTest.super.checkVectorMetrics(
+          "FindCommand", JsonApiMetricsConfig.SortType.NONE.name());
+      VectorSearchIntegrationTest.super.checkVectorMetrics(
+          "FindCommand", JsonApiMetricsConfig.SortType.SIMILARITY_SORT.name());
+      VectorSearchIntegrationTest.super.checkVectorMetrics(
+          "FindCommand", JsonApiMetricsConfig.SortType.SIMILARITY_SORT_WITH_FILTERS.name());
+      VectorSearchIntegrationTest.super.checkVectorMetrics(
```

Comment:
checkVectorMetrics could probably be `static` to avoid needing to reference via `super`?